### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by xiaopow**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def around
 
   def live_feed(postId)
     post = Post.find_by_id(postId)
-    return Post.order(taken_at: :desc).where("latitude > ?" , post.latitude - 0.001).where("latitude < ?" , post.latitude + 0.001).where("longitude > ?" , post.longitude - 0.001).where("longitude < ?" , post.longitude + 0.001).where("taken_at > ?", post.taken_at - 1.hour)
+    return Post.order(taken_at: :desc).where("latitude > ?" , post.latitude - 0.001).where("latitude < ?" , post.latitude + 0.001).where("longitude > ?" , post.longitude - 0.001).where("longitude < ?" , post.longitude + 0.001).where("taken_at > ?", post.taken_at - 1.hour).includes(:comments)
```
Comment (f3r) [inline app/controllers/posts_controller.rb]: perhaps its a good idea to create a couple of constants:

```
SEARCH_RADIUS = 0.001
SEARCH_TIME = 1.hour
```

So you can change it easily at the top of your controller and reuse it

File: config/routes.rb [hunk 2]
```diff
@@ -24,6 +24,7 @@
   root 'home#index'
   
   resources :posts
+  resources :comments
```
Comment (f3r) [inline config/routes.rb]:
``` ruby
resources :posts do
  resources :comments
end
```

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = current_user.comments.new(comment_params)
```
Comment (h8rry) [inline app/controllers/comments_controller.rb]: The variable should be `@comment`. Make sure you don't just take it from the notes.

File: app/controllers/comments_controller.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    @post.post_id = params[:post_id]
```
Comment (harrychenca) [inline app/controllers/comments_controller.rb]: You can perhaps add `post_id` into `comment_params`

File: app/assets/javascripts/comments.coffee [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/comments.coffee]: Can remove this file if you don't use it

File: app/assets/stylesheets/comments.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the comments controller here.
```
Comment (h8rry) [inline app/assets/stylesheets/comments.scss]: Can remove this file if you don't use it

File: app/controllers/posts_controller.rb [hunk 7]
```diff
@@ -1,25 +1,26 @@
...
+
+
   
   def around
     post = Post.find_by_id(params[:id])
     @posts = Post.where("latitude > ?" , post.latitude - 0.001).where("latitude < ?" , post.latitude + 0.001).where("longitude > ?" , post.longitude - 0.001).where("longitude < ?" , post.longitude + 0.001)
   end
 
-  def live_feed(postId)
+  def live_feed(postId,searchRadius)
```
Comment (h8rry) [inline app/controllers/posts_controller.rb]: There should be a space after comma after every single parameter.

**Comment by h8rry** [comment]
Looks good. Commit messages can be better though. :cocktail:

**Pull request merged**